### DISCUSSION TITLE
PharmacoSet Metadata Column Names

### DISCUSSION THREAD

**Discussion by ChristopherEeles** [bhklab/PharmacoGx#108] [PharmacoSet Standardization]
1. What are the existing standardized column names within the different slots of `PharmacoSet`?
2. What, if any, additional columns do we want to add as mandatory?
3. What, if any, additional optional metadata columns do we want to standardize?

**Comment by sisiranair** [comment]
4. Should the metadata be confined to lab curated annotations (AnnotationGx) OR contain metadata from authors? (Note: There can be common columns like drug or cell identifiers - Which one should be kept?)
